controllers/api/comment-routes.js

```diff
@@ -4,22 +4,57 @@ const { Comment, User, Post } = require('../../models');
 const withAuth = require('../../utils/auth');
 
 router.get('/', (req, res) => {
-    Comment.findAll()
+  Post.findAll({
```

Comment:
I don't think a Post.findAll is required here?

Reply:
I don't think so either, are you able to delete it when you review?